src/store/modules/DownloadModule.ts

```diff
@@ -7,7 +7,8 @@ interface DownloadProgress {
     assignId: number;
     initialMods: string[];
     modName: string;
-    progress: number;
+    downloadProgress: number;
+    installationProgress: number;
```

Comment:
Nitpick: "installProgress" would be shorter and match "download" better methinks.